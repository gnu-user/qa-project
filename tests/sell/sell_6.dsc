TITLE
    Sell Title Length (Long Title)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the sell 
    transaction. Enter a title which is 19 characters (max) and verify that it 
    is accepted.
DEPENDENCIES
    sell_4
