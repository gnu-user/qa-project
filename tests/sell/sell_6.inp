login
administrator
sell
The maximum length title.