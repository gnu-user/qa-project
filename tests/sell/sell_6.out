Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title: 
Enter the sale price in dollars: 